### PR TITLE
[EventHubs] Stress test using azure monitor service for metrics/traces/loggings

### PR DESCRIPTION
**Pull request by yunhaoling**
Addressing issue: https://github.com/Azure/azure-sdk-for-python/issues/18792

### CODE REVIEW

File: sdk/eventhub/azure-eventhub/stress/dev_requirement.txt
Comment (swathipil) [inline sdk/eventhub/azure-eventhub/stress/dev_requirement.txt:5]: Is this just for testing purposes right now? But will be removed after merging?

File: sdk/eventhub/azure-eventhub/stress/dev_requirement.txt
Comment (swathipil) [inline sdk/eventhub/azure-eventhub/stress/dev_requirement.txt:8]: is `opencensus.stats` part of the same package?

File: sdk/eventhub/azure-eventhub/stress/stress_runner.py
Comment (swathipil) [inline sdk/eventhub/azure-eventhub/stress/stress_runner.py:11]: unused imports?

**Review by swathipil — COMMENTED** [review]
left a few comments but o/w lgtm

File: sdk/eventhub/azure-eventhub/stress/dev_requirement.txt
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/stress/dev_requirement.txt:5]: actually this is on purpose to stick to sb 0.50.3 at least for current stress test implementation.
the sb v0.50.3 provides the ability to easily create new eventhub resources.

we might want to drop it off and move to using mgmt lib at some point, but that would be a separate issue&PR.

File: sdk/eventhub/azure-eventhub/stress/dev_requirement.txt
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/stress/dev_requirement.txt:8]: yep, opencensus-ext-azure depends on opencensus

File: sdk/eventhub/azure-eventhub/stress/stress_runner.py
Comment (yunhaoling) [inline sdk/eventhub/azure-eventhub/stress/stress_runner.py:11]:
```suggestion
```

Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/stress/stress_runner.py]: oops, good catch it was me trying to player with docker stuff adding some logic to deal with time.
removed

**Review by swathipil — APPROVED** [review]
approved (but you probably want to wait for Rakshith's review :P)

File: sdk/eventhub/azure-eventhub/stress/dev_requirement.txt
Comment (rakshith91) [inline sdk/eventhub/azure-eventhub/stress/dev_requirement.txt:8]: what version is being installed?

File: sdk/eventhub/azure-eventhub/stress/azure_eventhub_consumer_stress_async.py
Comment (rakshith91) [inline sdk/eventhub/azure-eventhub/stress/azure_eventhub_consumer_stress_async.py:94]: ASync?

File: sdk/eventhub/azure-eventhub/stress/dev_requirement.txt
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/stress/dev_requirement.txt:8]: it is 1.0.7

File: sdk/eventhub/azure-eventhub/stress/azure_eventhub_consumer_stress_async.py
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/stress/azure_eventhub_consumer_stress_async.py:94]: yep, this is the async consumer test file, so the name being "async"

File: sdk/eventhub/azure-eventhub/stress/logger.py
Comment (rakshith91) [inline sdk/eventhub/azure-eventhub/stress/logger.py:88]: let's make the env variable to APPLICATIONINSIGHTS_CONNECTION_STRING

- that way, we will not need to pass a connection string - OC exporter looks for the above env variable automatically ^^

File: sdk/eventhub/azure-eventhub/stress/azure_eventhub_consumer_stress_async.py
Comment (rakshith91) [inline sdk/eventhub/azure-eventhub/stress/azure_eventhub_consumer_stress_async.py:94]:
```suggestion
azure_metric_monitor = AzureMonitorMetric("Async EventHubConsumerClient", "Async EventHubConsumerClient")
```
do we need both name and description here? if yes, they should be different - i'd prefer to keep description optional

Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/stress/azure_eventhub_consumer_stress_async.py]: updated!

Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/stress/azure_eventhub_consumer_stress_async.py]: thanks for the suggestions! I thought it was mandatory because it's not optional in methods of opencensus.stats like `MeasureInt`, but apparently None is also allowed.

description is optional now

File: sdk/eventhub/azure-eventhub/stress/logger.py
Comment (yunhaoling) [reply sdk/eventhub/azure-eventhub/stress/logger.py:88]: thanks for the breadcrumb, I double check the code, yes, it's written this way.

**Pull request merged**
